Formula/cartridge-cli.rb

```diff
@@ -12,13 +12,28 @@ class CartridgeCli < Formula
     sha256 "6f579768ca73d8354418214ab79732a6b29a7f43dee8d3a4122bbab842906aeb" => :high_sierra
   end
 
-  depends_on "cmake" => :build
-  depends_on "tarantool"
+  depends_on "go" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args, "-DVERSION=#{version}"
-    system "make"
-    system "make", "install"
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    dir = buildpath/"src/github.com/tarantool/cartridge-cli"
+    dir.install buildpath.children
+
+    cd dir do
+      package = "github.com/tarantool/cartridge-cli/cli"
+      commit = Utils.safe_popen_read("git rev-parse --short HEAD").chomp
+      tag = Utils.safe_popen_read("git describe --tags").chomp
+
+      ldflags = [
+        "-s", "-w",
+        "-X #{package}/version.gitTag=#{tag}",
+        "-X #{package}/version.gitCommit=#{commit}"
+      ]
+
+      system "go", "mod", "download"
+      system "go", "build", "-o", bin/"cartridge", "-ldflags", ldflags.join(" "),
+          "github.com/tarantool/cartridge-cli/cli"
+    end
```

Comment:
We're working on `GOPATH` usage deprecation in homebrew-core (https://github.com/Homebrew/homebrew-core/issues/47627), so I believe we could simplify this installation like

```suggestion
    commit = Utils.safe_popen_read("git rev-parse --short HEAD").chomp

    ldflags = %W[
      -s -w
      -X github.com/tarantool/cartridge-cli/cli/version.gitTag=#{version}
      -X github.com/tarantool/cartridge-cli/cli/version.gitCommit=#{commit}
    ]

    system "go", "build", "-o", bin/"cartridge", "-ldflags", ldflags.join(" "), "cli/main.go"
```